src/Resource/Repository/RepositoryManager.js

```diff
@@ -0,0 +1,33 @@
+import createRepository from './RepositoryFactory';
+import Repository from './Repository';
+
+const repositories = new Map();
+
+const RepositoryManager = {};
+
+Object.assign(RepositoryManager, {
```

Comment:
Vanwaar de Object.assign?
Als we later iets willen toevoegen kunnen we dan de Object.assign gebruiken.

Reply:
Ik gebruik de RepositoryManager variable in een van de functies en daarom moet deze eerst gedefinieerd zijn. Ik vond dit de netste oplossing.
Je kunt later altijd nog een keer Object.assign doen als dat nodig is.